### PR TITLE
Conditionally Load Nova Packages

### PR DESCRIPTION
**Pull request by darkboywonder**
# Summary

This pull request addresses issue #241.
I would love a second pair of eyes to make this work cleaner if possible.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -86,6 +89,9 @@
         ],
         "post-create-project-cmd": [
             "@php artisan key:generate --ansi"
+        ],
+        "post-install-cmd": [
+            "[ $COMPOSER_DEV_MODE -eq 1 ] || composer require laravel/nova mrmonat/nova-translatable tightenco/nova-releases inspheric/nova-url-field tightenco/suggested-resources-shortcuts"
```
Comment (mattstauffer) [inline composer.json:94]: Can you tell me what's going on here?

Comment (darkboywonder) [reply composer.json:94]: I'm using `$COMPOSER_DEV_MODE` variable to check which environment we're building in. If it's local development, all nova packages are skipped and the programmer needs to manually require nova and all the additional packages it uses if they want to work locally with it. All nova packages are automatically installed if it's production. `$COMPOSER_DEV_MODE` is set by the `--no-dev` flag during the `composer install` call.

Comment (mattstauffer) [reply composer.json:94]: OHHHHH that's super clever. Thanks for explaining!!

**Comment by mattstauffer** [comment]
I don't see anything that needs cleaning up! Looks good to me

**Review by TammyTee — APPROVED** [review]

**Pull request merged**